saleor/graphql/product/sorters.py

```diff
@@ -0,0 +1,179 @@
+import graphene
+from django.db.models import Count, QuerySet
+
+from ...product.models import Category, Product
+from ..core.enums import OrderDirection
+from ..core.types import SortInputObjectType
+
+
+class AttributeSortField(graphene.Enum):
+    NAME = "name"
+    SLUG = "slug"
+    VALUE_REQUIRED = "value_required"
+    IS_VARIANT_ONLY = "is_variant_only"
+    VISIBLE_IN_STOREFRONT = "visible_in_storefront"
+    FILTERABLE_IN_STOREFRONT = "filterable_in_storefront"
+    FILTERABLE_IN_DASHBOARD = "filterable_in_dashboard"
+
+    DASHBOARD_VARIANT_POSITION = "dashboard_variant_position"
+    DASHBOARD_PRODUCT_POSITION = "dashboard_product_position"
+    STOREFRONT_SEARCH_POSITION = "storefront_search_position"
+    AVAILABLE_IN_GRID = "available_in_grid"
+
+    @property
+    def description(self):
+        # pylint: disable=no-member
+        descrtiptions = {
+            AttributeSortField.NAME.name: "name",
+            AttributeSortField.SLUG.name: "slug",
+            AttributeSortField.VALUE_REQUIRED.name: "the value required flag",
+            AttributeSortField.IS_VARIANT_ONLY.name: "the variant only flag",
+            AttributeSortField.VISIBLE_IN_STOREFRONT.name: (
+                "visibility in the storefront"
+            ),
+            AttributeSortField.FILTERABLE_IN_STOREFRONT.name: (
+                "the filterable in storefront flag"
+            ),
+            AttributeSortField.FILTERABLE_IN_DASHBOARD.name: (
+                "the filterable in dashboard flag"
+            ),
+            AttributeSortField.STOREFRONT_SEARCH_POSITION.name: (
+                "their position in storefront"
+            ),
+        }
+        if self.name in descrtiptions:
+            return f"Sort attributes by {descrtiptions[self.name]}."
+        if self == AttributeSortField.DASHBOARD_VARIANT_POSITION:
+            return "Sort variant attributes by their position in dashboard."
+        if self == AttributeSortField.DASHBOARD_PRODUCT_POSITION:
+            return "Sort product attributes by their position in dashboard."
```

Comment:
Will those messages be visible to the user? If so we need to address it's translation.